<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ATMECSINLT-601" failures="2" tests="12" name="com.atmecs.automating_travelsite.tests.HandlingWindow" time="15.050" errors="0" timestamp="2020-10-12T16:00:04 IST" skipped="10">
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="beforeTest" time="5.218">
    <failure message="expected [https://www.skyscanner.co.in/] but found [https://www.skyscanner.co.in/sttc/px/captcha-v2/index.html?url=Lz8=&amp;uuid=dec37330-0c75-11eb-a2b3-49929970d97f&amp;vid=]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.skyscanner.co.in/] but found [https://www.skyscanner.co.in/sttc/px/captcha-v2/index.html?url=Lz8=&uuid=dec37330-0c75-11eb-a2b3-49929970d97f&vid=]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.atmecs.automating_travelsite.basetest.BaseTest.beforeTest(BaseTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- beforeTest -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="handlingWindow" time="0.000">
    <skipped/>
  </testcase> <!-- handlingWindow -->
  <system-out/>
  <testcase classname="com.atmecs.automating_travelsite.tests.HandlingWindow" name="beforeTest" time="9.832">
    <failure message="expected [https://www.skyscanner.co.in/] but found [https://www.skyscanner.co.in/sttc/px/captcha-v2/index.html?url=Lz8=&amp;uuid=e2aa2340-0c75-11eb-832a-7d31b0ce91df&amp;vid=]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.skyscanner.co.in/] but found [https://www.skyscanner.co.in/sttc/px/captcha-v2/index.html?url=Lz8=&uuid=e2aa2340-0c75-11eb-832a-7d31b0ce91df&vid=]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.atmecs.automating_travelsite.basetest.BaseTest.beforeTest(BaseTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- beforeTest -->
  <system-out/>
</testsuite> <!-- com.atmecs.automating_travelsite.tests.HandlingWindow -->
